src/cosalib/vmware.py

```diff
@@ -68,6 +70,7 @@ def generate_ovf_parameters(self, vmdk, cpu=2,
         based on the qemu, vmdk, and info from the build metadata
         """
         disk_info = image_info(vmdk)
+        vmdk_size = str(os.stat(vmdk).st_size)
```

Comment:
Very minor but I'd do the `str` below since having it be a string is more a property of the OVF parameters than it is how we'd represent sizes internally in code.